addons/account/models/account_journal_dashboard.py

```diff
@@ -99,7 +99,7 @@ def build_graph_data(date, amount):
         locale = get_lang(self.env).code
 
         #starting point of the graph is the last statement
-        last_stmt = self._get_last_bank_statement(domain=[('move_id.state', '=', 'posted')])
+        last_stmt = self._get_last_bank_statement(domain=[('state', 'in', ['posted', 'confirmed'])])
```

Comment:
[the state is 'confirm'](https://github.com/odoo/odoo/blob/14.0/addons/account/models/account_bank_statement.py#L222) (which is weird, I agree)
```suggestion
        last_stmt = self._get_last_bank_statement(domain=[('state', 'in', ['posted', 'confirm'])])
```